src/api_authentication_app_skeleton/src/actions/api/sign_in/create.cr

```diff
@@ -0,0 +1,14 @@
+class Api::SignIn::Create < ApiAction
+  include Auth::RequireNotSignedIn
```

Comment:
Can you namespace all the API specific auth stuff in `Auth::Api`, example: `Auth::Api::RequireNotSignedIn`. That way they don't collide with the ones for browsers